content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,468 @@
+# How to build a task manager with angular.
+
+## Introduction
+
+If you are trying to learn a programming language or framework, it is good practice to build simple tasks which will go a long way to help solidify your knowledge in that language.  
+
+In this article, I will work you through building a simple task manager app (popularly known as a Todo app) using [angular](https://angular.io/docs) a JavaScript framework.
+
+
+## What is a Task Manager?
+
+A task manager is used to write, organize and rearrange tasks more efficiently. The task manager we are going to build will allow the user to add a task, mark the task as complete and also delete the task.
+
+
+## Key Takeaways
+1. How to create and structure an angular project.
+2. How to create modules and components in angular.
+3. Build a simple application.
+4. Save tasks to local storage
+
+
+
+## Prerequisite
+
+Before we get started, here are a few things to note:
+
+- You should have [Node.js](https://nodejs.org/en/) installed on your computer.
+- You should be fairly familiar with [Angular](https://angular.io/docs) and JavaScript.
+
+Knowledge of [TypeScript](https://www.typescriptlang.org/) is useful but not requisite.
+
+
+## Overview
+1. [Create a new angular project](https://www.dropbox.com/scl/fi/kzjnn0b7y1h3aw4p43wkx/How-to-build-a-task-manager-with-angular..paper?dl=0&rlkey=icow7dspxlq9l21h9m8q4m8sy#:uid=775660320270676740028602&h2=Create-a-new-angular-project)
+[](https://www.dropbox.com/scl/fi/kzjnn0b7y1h3aw4p43wkx/How-to-build-a-task-manager-with-angular..paper?dl=0&rlkey=icow7dspxlq9l21h9m8q4m8sy#:uid=017633763773070684372716&h2=Create-a-component)2. [Create a component](https://www.dropbox.com/scl/fi/kzjnn0b7y1h3aw4p43wkx/How-to-build-a-task-manager-with-angular..paper?dl=0&rlkey=icow7dspxlq9l21h9m8q4m8sy#:uid=017633763773070684372716&h2=Create-a-component)
+3. [Add task](https://www.dropbox.com/scl/fi/kzjnn0b7y1h3aw4p43wkx/How-to-build-a-task-manager-with-angular..paper?dl=0&rlkey=icow7dspxlq9l21h9m8q4m8sy#:uid=945266686904507102840148&h2=Add-Task-to-the-To-do-list)
+4. [Mark task as done](https://www.dropbox.com/scl/fi/kzjnn0b7y1h3aw4p43wkx/How-to-build-a-task-manager-with-angular..paper?dl=0&rlkey=icow7dspxlq9l21h9m8q4m8sy#:uid=829690951889208030685797&h2=Mark-Task-as-Done)
+5. [Delete task](https://www.dropbox.com/scl/fi/kzjnn0b7y1h3aw4p43wkx/How-to-build-a-task-manager-with-angular..paper?dl=0&rlkey=icow7dspxlq9l21h9m8q4m8sy#:uid=673533273713544444667587&h2=Delete-Task)
+6. [Save to local storage](https://www.dropbox.com/scl/fi/kzjnn0b7y1h3aw4p43wkx/How-to-build-a-task-manager-with-angular..paper?dl=0&rlkey=icow7dspxlq9l21h9m8q4m8sy#:uid=156966732915302471724639&h2=Save-task-to-local-storage)
```

Comment:
All attachments should reside on this article. Do not take us to another page to view these images. But am also not sure if we really need those images on top of the article?